packages/tools/pie-icons-react/package.json

```diff
@@ -59,10 +61,6 @@
     "rollup-plugin-typescript2": "0.34.1",
     "typescript": "5.3.2"
   },
-  "dependencies": {
-    "@justeattakeaway/pie-icons": "4.9.3",
-    "@justeattakeaway/pie-icons-configs": "4.5.1"
-  },
```

Comment:
these packages are only needed for the build step during development and the dis folder is just svg files.